### PR TITLE
support Fuse ds support onDelete updateStrategy for alluxioRuntime

### LINKED ISSUE #2368: [FEATURES] Fuse daemonset support onDelete updateStrategy for alluxioRuntime
**What feature you'd like to add:**

. Just like https://github.com/fluid-cloudnative/fluid/blob/master/charts/jindofsx/values.yaml#L60

**Why is this feature needed:**

### PR DESCRIPTION
**Pull request by hahchenchen**
Signed-off-by: sichen.zhao <sichen.zhao@outlook.com>

### Ⅰ. Describe what this PR does
support Fuse ds support onDelete updateStrategy for alluxioRuntime

### Ⅱ. Does this pull request fix one issue?
fix #2368 

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: charts/alluxio/values.yaml [hunk 1]
```diff
@@ -410,6 +410,8 @@ fuse:
   # - name: myConfigMapVolume
   #   mountPath: /data/configmap/myConfigMapVolume/
   volumes: []
+  updateStrategy:
```
Comment (cheyang) [inline charts/alluxio/values.yaml:413]: Please also update changelog and alluxio chart version. Thanks.

**Review by cheyang — CHANGES_REQUESTED** [review]

File: charts/alluxio/values.yaml (hunk 1 above)
Comment (hahchenchen) [reply charts/alluxio/values.yaml:413]: done

File: charts/alluxio/Chart.yaml [hunk 2]
```diff
@@ -23,3 +23,5 @@ maintainers:
   email: cheyang@163.com
 - name: Hao Ma
   email: allenhaozi@gmail.com
+- name: Sichen Zhao
+  email: sichen.zhao@outlook.com
```
[the remark this answers is not in the document]
Comment (hahchenchen) [reply charts/alluxio/Chart.yaml]: Ok, done.

**Review by cheyang — APPROVED** [review]

**Comment by cheyang** [comment]
thanks very much, @hahchenchen

**Pull request merged**